hawc/apps/vocab/migrations/0009_guideline_profile.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 5.1.1 on 2024-10-11 14:36
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("vocab", "0008_alter_term_namespace"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="GuidelineProfile",
+            fields=[
+                (
+                    "guideline_profile_id",
+                    models.PositiveIntegerField(
+                        primary_key=True, serialize=False, unique=True, verbose_name="ID"
+                    ),
+                ),
+                ("guideline_id", models.PositiveIntegerField()),
+                (
+                    "endpoint",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="vocab.term",
+                    ),
+                ),
+                ("endpoint_category", models.CharField(max_length=256)),
```

Comment:
Guideline_profile_id is the primary key, and I kept the 3 vocab name fields that make up the endpoint. If they were mainly for my benefit to load in the vocab ids, I can remove them!